matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/useUserSearchResults.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useWidgetApi } from '@matrix-widget-toolkit/react';
+import { isError } from 'lodash';
+import { useEffect, useState } from 'react';
+
+type SearchResults = Array<{
+  userId: string;
+  displayName?: string;
+  avatarUrl?: string;
+}>;
+
+export function useUserSearchResults(
```

Comment:
It might be useful to have tests for this component. I'm unsure if we really want to test the debounce since fakeTimer tests can become problematic, but we can try and at least check if the results and errors are properly returned. There should already be similar tests for other hooks that test e.g. the loading state etc.